### PR TITLE
You should hang yourself NOW! (abno free)

### PR DESCRIPTION
**Pull request by Litberries**
## About The Pull Request

Adds another noose

## Why It's Good For The Game

Jbird noose requires you to have and breach jbird, and have it kill a dude, so why not port the noose from bee! (also this ones animateeed)

## Changelog
:cl:
add: Added noose
/:cl:

### CODE REVIEW

File: code/game/objects/structures/noose.dm
Comment (LanceSmites328) [inline code/game/objects/structures/noose.dm:1]: A lot of the SPAN_X() have `</span>` , clean this up please

**Review by LanceSmites328 — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/noose.dm
Comment (Litberries) [reply code/game/objects/structures/noose.dm:1]: oh yeah my bad I forgot to change that

**Comment by StrangeUser** [comment]
funny

**Comment by Litberries** [comment]
I opened a new PR for this

**Pull request closed without merging**